atlassian_jwt_auth/aio/verifier.py

```diff
@@ -0,0 +1,24 @@
+import asyncio
+
+from atlassian_jwt_auth.verifier import JWTAuthVerifier as _JWTAuthVerifier
+
+
+class JWTAuthVerifier(_JWTAuthVerifier):
+    def __init__(self, public_key_retriever, *, loop=None, **kwargs):
+        if loop is None:
+            loop = asyncio.get_event_loop()
+        self.loop = loop
```

Comment:
There's no reason for keeping this loop attribute in the class unless you actually use it somewhere.